docker/redmine/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:7
+MAINTAINER Eric Helms
+
+RUN yum -y install vim ruby ruby-devel gcc make libxml2-devel libxslt-devel postgresql-devel libvirt-devel wget git mysql-devel ImageMagick-devel
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+RUN \curl -sSL https://get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm install 1.9.3"
```

Comment:
I would have to double check, I feel like I was hitting issues with a newer version.